### PR TITLE
Updated MultidbConfigDialog

### PR DESCRIPTION
**Pull request by Priyanka328**
- Now user can pick the main_server from a combobox
- User can choose to lookup in main_server besides import_servers

### CODE REVIEW

File: src/MultiDBConfigDialog.cpp [hunk 1]
```diff
@@ -267,56 +282,81 @@ namespace xrock_gui_model
...
             else
-                backend["path"] = w.urlOrPath.toStdString();
-            backend["graph"] = w.graph.toStdString();
+                backend["path"] = b.urlOrPath.toStdString();
+            backend["graph"] = b.graph.toStdString();
             config["import_servers"].push_back(std::move(backend));
         }
-
+ 
+        config["also_lookup_main_server"] = cbLookupInMainDatabase->isChecked();
```
Comment (m0rsch1) [inline src/MultiDBConfigDialog.cpp:317]: This flag is not available in XDBI

Comment (Priyanka328) [reply src/MultiDBConfigDialog.cpp:317]: yes. as we agreed to not to have this flag , I will remove it.

**Comment by m0rsch1** [comment]
When i select a bundle, the MultiDBConfig Dialog is not populated with the values of the current configuration but with a different one.

**Comment by Priyanka328** [comment]
> Although this might not be the right spot i get the following messages on the console when having loaded a model and moving the mouse:
> 
> ```
> hover check: 1234-1242 / 1217-1225
> hover check: 1234-1242 / 1188.2-1196.2
> out hover check: 1490.94-1498.94 / 1217-1225
> out hover check: 1490.94-1498.94 / 1188.2-1196.2
> ```
 this is resolved

**Comment by Priyanka328** [comment]
> The main server is in the list twice ... at the top highlighted (as main server) and again at the end of the list. Is this normal or is my config bogus?
> 
> I expected to have every distinct database in the list once and the main database highlighted.

this behavior is so that user also visually notice that what is the checkbox is`AlsoLoopUpInMainServerCheckbox` is doing. Yeah. we can hide that ofcourse .

**Pull request merged**